pusher/apns.go

```diff
@@ -125,9 +124,8 @@ func (a *APNSPusher) configure(queue interfaces.APNSPushQueue, db interfaces.DB,
 				"game":   k,
 			}).Error(err)
```

Comment:
I think that adding a better error message will make easier to locate this log and identify which games are failing. What do you think?

```suggestion
			}).WithError(err).Error("failed to initialize game")
```